lib/modules/manager/terragrunt/readme.md

```diff
@@ -1,3 +1,5 @@
+Extracts dependencies from `terragrunt.hcl` files, and the associated `.terraform.lock.hcl` files too.
```

Comment:
this seems wrong, as we don't extract the lock files?

Reply:
Ah, thanks for the correction. I'm still learning what the terms and processes are in this project.